catalog/core/catalog-core-validationparser/src/test/groovy/ddf/catalog/validation/impl/ValidationParserSpecTest.groovy

```diff
@@ -593,4 +649,46 @@ class ValidationParserSpecTest extends Specification {
     }
 }
 '''
+
+    String metacardValidator = '''
+{
+  "metacardvalidators": [{
+    "fallback.common": [{
+        "validator": "requiredattributes",
+        "requiredattributes": [
```

Comment:
Should we make this key similiar to the other validators we have? such that we have the arguments for the validator all under an `arguments` key. That way it wouldn't have to change for each validator (Also `requiredattributes` might not make sense for other arbitrary metacard validators arguments)

So it would be similiar to this: https://github.com/codice/ddf/blob/master/catalog/core/catalog-core-validationparser/src/test/groovy/ddf/catalog/validation/impl/ValidationParserSpecTest.groovy#L589-L590